### PR TITLE
VersionMessage: migrate `time` field to `java.time` API

### PR DESCRIPTION
**Pull request by schildbach**

### CODE REVIEW

File: core/src/main/java/org/bitcoinj/core/Peer.java [hunk 1]
```diff
@@ -388,8 +388,7 @@ public String toString() {
         String servicesStr = Strings.emptyToNull(VersionMessage.toStringServices(vPeerVersionMessage.localServices));
         helper.add("services",
                 vPeerVersionMessage.localServices + (servicesStr != null ? " (" + servicesStr + ")" : ""));
-        long peerTime = vPeerVersionMessage.time * 1000;
-        helper.add("time", String.format(Locale.US, "%tF %tT", peerTime, peerTime));
+        helper.add("time", TimeUtils.dateTimeFormat(vPeerVersionMessage.time .toEpochMilli()));
```
Comment (msgilligan) [inline core/src/main/java/org/bitcoinj/core/Peer.java:391]: Looks like there's an unwanted space before `.toEpachMilli()`

**Review by msgilligan — CHANGES_REQUESTED** [review]
One minor whitespace issue that I think you'll want to fix.

File: core/src/main/java/org/bitcoinj/core/Peer.java (hunk 1 above)
Comment (schildbach) [reply core/src/main/java/org/bitcoinj/core/Peer.java:391]: Fixed.

**Comment by schildbach** [comment]
Merged.

**Pull request closed without merging**